recipes/curl/meta.yaml

```diff
@@ -19,10 +20,12 @@ requirements:
   build:
     # python is a build requirement on Windows to resolve features.
     - python    # [win]
+    - pkg-config  # [unix]
     - openssl   # [unix]
     - zlib
   run:
     - openssl   # [unix]
+    - certifi   # [unix]
```

Comment:
It doesn't care that the file doesn't exist.  It just tells it where to look once it goes looking (at runtime)